web-bluetooth/link-loss.js

```diff
@@ -0,0 +1,70 @@
+var alertLevelCharacteristic;
+
+function onReadButtonClick() {
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice(
+    {filters: anyDevice(), optionalServices: ['link_loss']})
+  .then(device => {
+    log('Connecting to GATT Server...');
+    return device.gatt.connect();
+  })
+  .then(server => {
+    log('Getting Link Loss Service...');
+    return server.getPrimaryService('link_loss');
+  })
+  .then(service => {
+    log('Getting Alert Level Characteristic...');
+    return service.getCharacteristic('alert_level');
+  })
+  .then(characteristic => {
+    alertLevelCharacteristic = characteristic;
+    document.querySelector('#writeButton').disabled = false;
+    log('Reading Alert Level...');
+    return characteristic.readValue();
+  })
+  .then(value => {
+    log('> Alert Level: ' + getAlertLevel(value));
+  })
+  .catch(error => {
+    document.querySelector('#writeButton').disabled = true;
+    log('Argh! ' + error);
+  });
+}
+
+function onWriteButtonClick() {
+  if (!alertLevelCharacteristic) {
+    return;
+  }
+  log('Setting Alert Level...');
+  let value = [document.querySelector('#alertLevelValue').value];
+  alertLevelCharacteristic.writeValue(new Uint8Array([value]))
```

Comment:
This builds a doubly-nested array: `new Uint8Array([[thevalue]])` ... which works for a single-value array somehow, but is pretty confusing.